client/signup/steps/domains/index.jsx

```diff
@@ -514,8 +514,9 @@ class DomainsStep extends React.Component {
 			return null;
 		}
 
-		const { translate } = this.props;
-		let backUrl = undefined;
+		const { translate, selectedSite } = this.props;
+		let backUrl = undefined,
+			backLabelText = null;
```

Comment:
Do we need to specify values here?

`let backUrl, backLabelText;`

`backLabelText` has a [default value](https://github.com/Automattic/wp-calypso/blob/master/client/signup/navigation-link/index.jsx#L42
) of `''` anyway.

Reply:
Implemented in [cbb4f6f](https://github.com/Automattic/wp-calypso/pull/31235/commits/cbb4f6fa614e6d2a3c44b688d615cf630cbd5d6c).